cloud/azure/modules/app/main.tf

```diff
@@ -12,6 +18,47 @@ resource "azurerm_virtual_network" "civiform_vnet" {
   address_space       = var.vnet_address_space
 }
 
+resource "azurerm_subnet" "storage_subnet" {
+  name                                           = "storage-subnet"
+  resource_group_name                            = azurerm_resource_group.rg.name
+  virtual_network_name                           = azurerm_virtual_network.civiform_vnet.name
+  address_prefixes                               = ["10.0.8.0/24"]
+  service_endpoints                              = ["Microsoft.Storage"]
+  enforce_private_link_endpoint_network_policies = true
+}
+
+resource "azurerm_storage_account" "file_storage" {
+  name                = "${var.application_name}${random_string.resource_code.result}"
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+data "http" "myip" {
+  url = "https://ipv4.icanhazip.com"
+}
+
+# adding the bypass/ip_rules is a workaround for azure's firewall
+# if we don't add this we can't add files to this container
```

Comment:
> can't add files to this container
what does it mean to add files to the container?

Reply:
yeah good point let me do a lil renaming here. i couldn't add the container (which i think is like a bucket in s3) to the storage account